cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -59,20 +59,20 @@ func extractExecutionState(
 	reporters := []ledger.Reporter{}
 
 	if migrate {
-		storageFormatV6Migration := mgr.StorageFormatV6Migration{
-			Log:       log,
-			OutputDir: outputDir,
-		}
+		// storageFormatV6Migration := mgr.StorageFormatV6Migration{
+		// 	Log:       log,
+		// 	OutputDir: outputDir,
+		// }
 
-		storageUsedUpdateMigration := mgr.StorageUsedUpdateMigration{
-			Log:       log,
-			OutputDir: outputDir,
-		}
+		// storageUsedUpdateMigration := mgr.StorageUsedUpdateMigration{
+		// 	Log:       log,
+		// 	OutputDir: outputDir,
+		// }
 
 		migrations = []ledger.Migration{
 			mgr.PruneMigration,
```

Comment:
If there is no migration, do we still need to prune?

Reply:
yeah pruning helps to prune nil values in the trie, so it reduces the checkpointer size and memory usage, we should always prune until we implement a service to prune regularly.